attributes/default.rb

```diff
@@ -339,10 +339,14 @@
 default['cfncluster']['lustre']['base_url'] = value_for_platform(
   'centos' => {
     '>=8' => "https://fsx-lustre-client-repo.s3.amazonaws.com/el/8/x86_64/",
-    'default' => "https://fsx-lustre-client-repo.s3.amazonaws.com/el/7.#{get_rhel7_kernel_minor_version}/x86_64/"
+    '7.8' => "https://fsx-lustre-client-repo.s3.amazonaws.com/el/7.8/x86_64/",
```

Comment:
isn't better leave the default as it was and move the check with the `if node['platform'] == 'centos'` inside the get_rhel7_kernel_minor_version function?

Reply:
Done